docs/simulate/pauli_observables.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<p>{:external} doesn't work in jupyter notebooks. (I confirmed this as we have a place where this is tried and it failed https://quantumai.google/cirq/simulate/params ... I'll fix those)</p><p> </p><p>"be represented by a tensor product of linear combinations of the Pauli operators. "</p><p>should be</p><p>"be represented by a  linear combinations of tensor products of Pauli operators."</p><p>(i.e. gate like 1/sqrt(2) (II + i XX) is a linear combination of tensor products of Pauli, but cannot be written as a tensor product of linear combinations of Paulis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.